pkg/plugin/sdk/protocol/protocol.proto

```
@@ -0,0 +1,156 @@
+syntax = "proto3";
+
+package gameap.plugin.sdk.protocol;
```

Buf (1.71.0)

[error] Files with package "gameap.plugin.sdk.protocol" must be within a directory "gameap/plugin/sdk/protocol" relative to root but were in directory "pkg/plugin/sdk/protocol".

(PACKAGE_DIRECTORY_MATCH)